dlt/sources/sql_database/helpers.py

```diff
@@ -96,9 +100,15 @@ def _make_query(self) -> SelectAny:
             return query  # type: ignore[no-any-return]
 
         if self.last_value is not None:
-            query = query.where(filter_op(self.cursor_column, self.last_value))
+            where_and_clauses = [filter_op(self.cursor_column, self.last_value)]
```

Comment:
queries are OK. should `exclude` option be independent from `if self.last_value is not None:`? I think so. we could skip NONE also on initial load